Cargo.toml

```
@@ -12,16 +12,17 @@ exclude = [".github/", ".cirrus.yml", "tests/",  "test_data/", "contrib/", "pypr
 [[bin]]
 name = "lianad"
 path = "src/bin/daemon.rs"
-required-features = ["jsonrpc_server"]
+required-features = ["jsonrpc_server", "daemon"]
 
 [[bin]]
 name = "liana-cli"
 path = "src/bin/cli.rs"
 required-features = ["jsonrpc_server"]
 
 [features]
-default = ["jsonrpc_server"]
+default = ["jsonrpc_server", "daemon"]
 jsonrpc_server = []
+daemon = ["libc"]
```

I think it would be cleaner to get rid of the `jsonrpc_server` feature then. It implies `daemon` (and `daemon` implies it).

```suggestion
default = ["daemon"]
daemon = ["libc"]
```

---

And also the dependency for `jsonrpc_server` is wrong: it needs libc.